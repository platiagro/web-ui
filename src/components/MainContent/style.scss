.main-content {
  padding: 20px 50px;
}

.main-content .content {
  background: #fff;
  padding: 24px;
  height: 75vh;
  display: flex;
  align-items: center;
  justify-content: center;
}
