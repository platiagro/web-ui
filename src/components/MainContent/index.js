import React from 'react';

import './style.scss';

import { Layout } from 'antd';

import ContentHeader from '../ContentHeader';

const { Content } = Layout;

const MainContent = () => (
  <div>
    <ContentHeader />
    <Content className='main-content'>
      <div className='content' />
    </Content>
  </div>
);

export default MainContent;
