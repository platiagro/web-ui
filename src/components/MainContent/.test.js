import React from 'react';

import { shallow } from 'enzyme';

import MainContent from '.';

describe('MainContent component', () => {
  it('renders without crashing', () => {
    shallow(<MainContent />);
  });

  it('renders html correctly', () => {
    const mainContentShalowed = shallow(<MainContent />);

    expect(mainContentShalowed).toMatchSnapshot();
  });
});
