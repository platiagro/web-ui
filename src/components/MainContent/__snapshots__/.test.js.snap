// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent component renders html correctly 1`] = `
<div>
  <ContentHeader
    subTitle=""
    title=""
  />
  <Adapter
    className="main-content"
  >
    <div
      className="content"
    />
  </Adapter>
</div>
`;
